config/initializers/apipie.rb

```diff
@@ -15,6 +15,7 @@
   config.default_locale = FastGettext.default_locale
   config.locale = ->(loc) { loc ? FastGettext.set_locale(loc) : FastGettext.locale }
 
+  ActiveRecord::Migration.maintain_test_schema!
```

Comment:
Hmm, it looks like it's loaded at some other point before db initialization as well (not on any initializer) - I checked and putting it in a lambda will not make it call during initialization time, but it'll still be called early.

In any case maybe a good solution is to call maintain_test_schema in a separate, earlier initializer?